pkg/metrics/metrics.go

```diff
@@ -44,12 +46,18 @@ func NewMetrics() Metrics {
 			Help: "completed pipelines",
 		},
 			[]string{"result"}),
-		Duration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
-			Name:    "steward_pipelinerun_duration_seconds",
-			Help:    "pipeline run durations",
+		TotalDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
Revert to former name:
```suggestion
		Duration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
```